src/eager/setup.py

```diff
@@ -136,20 +177,24 @@ def gen_ort_aten_ops():
   gen_cpp_name = "ort_aten.g.cpp"
   if os.path.exists(gen_cpp_name):
     os.remove(gen_cpp_name)
-  subprocess.check_call([
-    python_exe,
-    os.path.join(self_dir, 'opgen', 'opgen.py'),
-    gen_cpp_name
-  ])
+  cmd = [python_exe, os.path.join(self_dir, 'opgen', 'opgen.py'), '--output_file', gen_cpp_name]
+  if args.use_preinstalled_torch:
+    cmd.append('--use_preinstalled_torch')
+  subprocess.check_call(cmd)
 
-if os.path.isfile(pytorch_compile_commands_path):
-  print('Skipping PyTorch Build (remove compile_commands.json to build it):')
-  print(f'  {pytorch_compile_commands_path}')
-else:
-  print('Building PyTorch...')
-  build_pytorch()
 
-build_ort()
+if not args.use_preinstalled_torch:
+  if os.path.isfile(pytorch_compile_commands_path):
+    print('Skipping PyTorch Build (remove compile_commands.json to build it):')
+    print(f'  {pytorch_compile_commands_path}')
```

Comment:
```suggestion
    print('Skipping PyTorch Build (remove compile_commands.json to build it)')
```
Maybe remove debug prints?

Reply:
I left these original prints as-is, thinking they are useful. For example, this particular case skips building torch even when the instruction is to build torch because a previous iteration has built torch. 

Let me know what you think.